shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -492,10 +528,32 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
 
   SemanticsObject* root = objects_.get()[@(kRootNodeId)];
 
+  bool routeChanged = false;
+  SemanticsObject* lastAdded = nil;
+
   if (root) {
     if (!view_.accessibilityElements) {
       view_.accessibilityElements = @[ [root accessibilityContainer] ];
     }
+    NSMutableArray<SemanticsObject*>* newRoutes = [[[NSMutableArray alloc] init] autorelease];
+    [root collectRoutes:newRoutes];
+    for (SemanticsObject* route in newRoutes) {
```

Comment:
Hi @jonahwilliams, do you remember why we check any new route in the  previous_routes_  here? This will make the it to not announce route change when any new route is in the previous routes

for example [Route1]  -> [Route1, Route2] will not announce any route.